<?xml version="1.0"?>
<launch>
    <include file="$(find droplet_underwater_assembly)/apm.launch" />
    <include file="$(find mini_ahrs_ros)/launch/mini_ahrs_standalone.launch"/>
    <include file="$(find exposure_control)/launch/live_robot_stag.launch"/>
    <node name="forward_stag_to_breadcrumb" pkg="localization_informed_planning_sim" type="forward_stag_to_breadcrumb.py" required="true" output="screen"/>

    <node name="controller" pkg="localization_informed_planning_sim" type="global_position_controller.py" required="true" output="screen">
      <param name="simulation_mode" type="bool" value="false"/>
      <param name="dry_run_mode" type="bool" value="false"/>
    </node>

    <node name="assembly_manager" pkg="localization_informed_planning_sim" type="assembly_process_manager.py" required="true" output="screen">
        <param name="use_controller_test_plan" value="true"/>
    </node>
</launch>
